PokeAlarm/Manager.py

```diff
@@ -658,6 +658,7 @@ def process_gym(self, gym):
             "dist": get_dist_as_str(dist),
             'dir': get_cardinal_dir([lat, lng], self.__latlng),
             'new_team': cur_team,
+            'new_team_id': "Team" + str(to_team_id),
```

Comment:
Can we get this changed to `"team{}".format(to_team_id)`

(mostly to match how we do other string substitutions)